nx-dev/ui-terms/src/lib/header.tsx

```diff
@@ -0,0 +1,48 @@
+import { SectionHeading, SectionDescription } from '@nx/nx-dev/ui-common';
+import Link from 'next/link';
+
+export function Header() {
+  return (
+    <header className="mx-auto max-w-prose">
+      <div className="text-center">
+        <SectionHeading as="h2" variant="title">
+          Nx Terms of Service
+        </SectionHeading>
+        <SectionHeading as="p" variant="subtitle" className="mt-6">
+          Effective July 2, 2024 (Version 4.0)
+        </SectionHeading>
+      </div>
+
+      <SectionDescription as="p" className="mt-12">
+        Welcome to the Nx website located at https://nx.app/ (the &quot;
```

Comment:
@juristr pending your confirmation
```suggestion
        Welcome to the Nx website located at https://nx.dev/ (the &quot;
```